go/v1beta1/storage/elasticsearch.go

```diff
@@ -383,78 +382,126 @@ func (es *ElasticsearchStorage) CreateOccurrence(ctx context.Context, projectID,
 		o.CreateTime = ptypes.TimestampNow()
 	}
 
-	m := jsonpb.Marshaler{}
-	str, err := m.MarshalToString(o)
+	str, err := protojson.Marshal(proto.MessageV2(o))
 	if err != nil {
 		return nil, createError(log, "error marshalling occurrence to json", err)
 	}
 
+	occurrenceIndex := fmt.Sprintf("%s-%s-%s", indexPrefix, projectID, "occurrences")
```

Comment:
We are creating this index name in a few places. Should this be a function to make sure it is consistent?